helm-charts/yunikorn-all/Chart.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+appVersion: "1.0"
+description: A Helm chart for YuniKorn
```

Comment:
the description should just explain what this is. So suggest rephrasing it to something like "YuniKorn scheduler for Kubernetes"